src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/table.svelte

```diff
@@ -12,13 +13,34 @@
         TableRowLink,
         TableScroll
     } from '$lib/elements/table';
+    import { organization } from '$lib/stores/organization';
+    import { teamPrefs } from '$lib/stores/team';
+    import type { Models } from '@appwrite.io/console';
+    import { onMount } from 'svelte';
     import type { PageData } from './$types';
-    import { collection, columns } from './store';
+    import type Attribute from './document-[document]/attribute.svelte';
+    import RelationshipsModal from './relationshipsModal.svelte';
+    import { attributes, collection, columns } from './store';
 
     export let data: PageData;
 
     const projectId = $page.params.project;
     const databaseId = $page.params.database;
+    let showRelationships = false;
+    let selectedRelationship: Models.AttributeRelationship = null;
+    let relationshipData: [];
+
+    onMount(() => {
+        teamPrefs.load($organization.$id);
+        columns.set([
+            ...$collection.attributes.map((attribute) => ({
+                id: attribute.key,
+                title: attribute.key,
+                type: attribute.type,
+                show: true
+            }))
```

Comment:
Can we include `$createdAt` and `$updatedAt` to this too for https://github.com/appwrite/appwrite/issues/3592?

Reply:
@chenparnasa @sara-k-48  what do you think?